docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  python:
+    build:
+      context: .
+      dockerfile: ./python/Dockerfile
```

Comment:
if we want support several languages it seems docker-compose for python have to be placed in python folder

Reply:
We can add move languages right here (?)